priv/static/thesis-editor.js

```diff
@@ -28078,7 +28078,472 @@ exports.default = EditButton;
 
```

Comment:
Skip this file and jump to https://github.com/infinitered/thesis-phoenix/pull/28/files#diff-a433b9efa3775042bce9c87867f7fe89